e2e/process-services-cloud/task_filters_cloud.e2e.ts

```diff
@@ -88,15 +92,22 @@ describe('Task filters cloud', () => {
 
             tasksCloudDemoPage.completedTasksFilter().clickTaskFilter();
             expect(tasksCloudDemoPage.checkActiveFilterActive()).toBe('Completed Tasks');
+
             tasksCloudDemoPage.taskListCloudComponent().getDataTable().checkContentIsDisplayed(completedTask);
         });
 
-        xit('[C289957] Should display task filter results when task filter is selected', async () => {
+        it('[C289957] Should display task filter results when task filter is selected', async () => {
             await tasksService.init(user, password);
-            let task = await tasksService.createStandaloneTask(myTask, simpleApp);
+            await tasksService.createStandaloneTask(myTask, simpleApp);
 
             tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
             expect(tasksCloudDemoPage.checkActiveFilterActive()).toBe('My Tasks');
+
+            tasksCloudDemoPage.customFilter().clickCustomFilters();
```

Comment:
Lets remove these steps of selecting a custom filter by just claiming the task in the preconditions.